File: deaf_grandma.py
def deaf_grandma():
    goodbye_count = 0
    
    print("HEY KID!")

    while(goodbye_count < 2):
        x = input('> ')

        if(x == ""):
            print("WHAT?!")
        else:
            if(x.isupper() == False):
                print("SPEAK UP, KID!")
            elif(x == 'GOODBYE!'):
                if(goodbye_count < 1):
                    print("LEAVING SO SOON?")
                    goodbye_count += 1
                else:
                    print("LATER, SKATER!")
                    goodbye_count += 1
            else:
                print("NO, NOT SINCE 1946!")




deaf_grandma()

File: deaf_grandma.js
function deafGrandma() {
    let goodbye_count = 0;  

    console.log("HEY KID!");

    while(goodbye_count < 2) {  
        const prompt = require("prompt-sync")({ sigint: true });    // Allows node to get user input from terminal
        const x = prompt("> "); // Prompts the user for an input

        if(x === "") { 
            console.log("WHAT?!"); 
        } else {
            if(x !== x.toUpperCase()) { 
                console.log("SPEAK UP, KID!"); 
            } else if (x === "GOODBYE!") {  
                if(goodbye_count < 1) { 
                    console.log("LEAVING SO SOON?");    
                    goodbye_count++;    
                } else {   
                    console.log("LATER, SKATER!");  
                    goodbye_count++;   
                }
            } else {   
                console.log("NO, NOT SINCE 1946!"); 
            }
        }

    }

}

deafGrandma();